README.md

```diff
@@ -1,4 +1,31 @@
 # Kubernetes Metrics Server
 
-more details TBD
+## User documentation
 
+User documentation can be found in
+[the official Kubernetes documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/core-metrics-pipeline/).
```

Comment:
Maybe `You can find the user guide in [the official Kubernetes documentation][...].`

Reply:
done